inc/alert.class.php

```diff
@@ -567,11 +455,140 @@ static function displayAlerts($params = []) {
       }
    }
 
-   static function getTypes() {
-      return [self::GENERAL => __("General", 'news'),
-              self::INFO    => __("Information", 'news'),
-              self::WARNING => __("Warning", 'news'),
-              self::PROBLEM => __("Problem", 'news')];
+   /**
+    * Compute alert size classes
+    *
+    * @param string $size Alert size
+    *
+    * @return string Bootstrap col classes
+    */
+   public static function getSizeClasses(string $size): string {
+      switch ($size) {
+         case self::SMALL:
+            return "col-xxl-4 col-xl-4 col-12";
+
+         default:
+         case self::MEDIUM:
+            return "col-xxl-6 col-xl-6 col-12";
+
+         case self::BIG:
+            return "col-xxl-8 col-xl-8 col-12";
+
+         case self::MAXIMUM:
+            return "col-12";
+      }
+   }
+
+   public static function displayAlert($alert, $p)
+   {
+      // Clean unexpected \r\n values (is there a better way to prevent this ?)
+      $alert['message'] = str_replace('\r\n', "", $alert['message']);
```

Comment:
How did you get these values ?

Reply:
I think the old code was not using the Sanitizer class, seem fixed after using it (500aaa3).